### PR TITLE
[readability-js] Increase visibility of certain elements under darkmode

### PR DESCRIPTION
**Pull request by rien333**
I've noticed significant differences between Qt versions in the color output produced under darkmode (often improvements!). Per Qt 5.15.2, I noticed that lines/borders drawn in dark colors are  less visible on pages produced by `readability-js` (not entirely sure why — a lot of borders on other pages seem to stand out more). In fact, table borders are currently invisible, because they're drawn in black, which for some reason doesn't get inverted anymore. 

To rectifiy this, I decided to draw all borders/lines in grey, in line with  what chromium's (?) user agent stylesheet seems to be doing. Because a lot of borders on the web actually seem to be drawn in grayish colors (and not in pure black/white), I think this is a good permanent change (and not just a workaround for something in Qt 5.15.2).

### CODE REVIEW

**Comment by rien333** [comment]
tldr; Qt 5.15.2 currently makes certain web page elements invisible within readability-js, fixed by explicitly setting their color to gray (which matches how chromium renders these elements in its default stylesheet). 

@The-Compiler any problems with this tiny change?

**Comment by The-Compiler** [comment]
No issues from what I can see, I've just been working on different stuff for the past couple of days - looking at the new PRs as we speak, so this will most likely be merged today.

**Comment by The-Compiler** [comment]
Thanks!
